src/toil/utils/toilStatus.py

```diff
@@ -184,21 +188,29 @@ def getStatus(jobStoreName):
         """
         Determine the status of a workflow.
 
-        Checks for the existance of files created in the toil.Leader.run(). In toil.Leader.run(), if a workflow completes
+        If the jobstore does not exist, this returns 'QUEUED', assuming it has not been created yet.
+
+        Checks for the existence of files created in the toil.Leader.run(). In toil.Leader.run(), if a workflow completes
         with failed jobs, 'failed.log' is created, otherwise 'succeeded.log' is written. If neither of these exist,
         the leader is still running jobs.
 
-        :return: A string indicating the status of the workflow. ['COMPLETED','RUNNING','ERROR']
+        :return: A string indicating the status of the workflow. ['COMPLETED', 'RUNNING', 'ERROR', 'QUEUED']
         :rtype: str
         """
-        jobstore = Toil.resumeJobStore(jobStoreName)
+        try:
+            jobstore = Toil.resumeJobStore(jobStoreName)
+        except NoSuchJobStoreException:
+            return 'QUEUED'
+
         try:
             with jobstore.readSharedFileStream('succeeded.log') as successful:
-                return 'COMPLETED'
+                pass
+            return 'COMPLETED'
         except NoSuchFileException:
             try:
-                with jobstore.readSharedFileStream('failed.log') as failedLog:
-                    return 'ERROR'
+                with jobstore.readSharedFileStream('failed.log') as failed:
```

Comment:
Sorry, I know this isn't relevant to this PR, but I just noticed: is `failed.log` ever actually deleted on resume? I can't see anywhere that the deletion happens. If not, a failed but successfully restarted workflow will forever get the FAILED status (until it completes).

Reply:
That's a really good point.  I'll create an issue and a corresponding PR.  Thanks for catching this!